.changeset/clever-pillows-sparkle.md

```diff
@@ -0,0 +1,6 @@
+---
+'@itwin/itwinui-layouts-css': minor
+'@itwin/itwinui-layouts-react': minor
+---
+
+Layout for sticky Bottom Bar slot and Action Bar within Bottom Bar. Action Bar has 3 slots for custom components: `ActionBar.Left`, `ActionBar.Center` and `ActionBar.Right`
```

Comment:
maybe add a small example code snippet (3-5 lines)

Reply:
Updated. How's this? Maybe slightly long.

Reply:
seems fine